prime-dotnet-webapi/Controllers/EnrolleePaperSubmissionsController.cs

```diff
@@ -258,5 +268,72 @@ public async Task<ActionResult> FinalizeEnrolleePaperSubmission(int enrolleeId)
 
             return Ok();
         }
+
+        // HEAD: api/Enrollees/potential-paper-enrollee/5
+        /// <summary>
+        /// Gets all paper enrollees and checks whether or not there is a match in dateOfBirth with the current logged on enrollee.
+        /// </summary>
+        [HttpHead("potential-paper-enrollee/{dateOfBirth}")]
+        [Authorize(Roles = Roles.TriageEnrollee + "," + Roles.PrimeEnrollee)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
```

Comment:
very minor point here: we like to put the annotations in the following order:
```c#
[HttpMethod]
[Authorize]
[Error Responses]
[Success Responses]
```
so that the response type is right above the method signature

Reply:
Fixed